src/Panel/Menu.php

```diff
@@ -127,7 +127,9 @@ public function entry(array $area): array|false
 			'link'     => $area['link'] ?? null,
 			'dialog'   => $area['dialog'] ?? null,
 			'drawer'   => $area['drawer'] ?? null,
+			'target'   => $area['target'] ?? null,
 			'text'     => I18n::translate($area['label'], $area['label']),
+			'title'    => $area['title'] ?? null,
```

Comment:
Suggestion: We should use I18n::translate for the title

Reply:
@bastianallgeier added it